arbitrum-docs/launch-orbit-chain/how-tos/customize-stf.mdx

```diff
@@ -140,13 +174,33 @@ Once you have the WASM module root, you can put it on-chain by calling `setWasmM
 The rollup contract address can be found in the chain deployment info JSON.
 You can confirm that the WASM module root was updated by calling `wasmModuleRoot()` on the rollup contract.
 
-Once you have set the new WASM module root on-chain, you may then run the new node image with:
+Once you have set the new WASM module root on-chain, you may then run the new node image. Following a similar process than before, we will first remove the parameter `--node.staker.dangerous.without-block-validator` to enable fraud proofs back.
 
-```shell
-docker run --rm -it -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 custom-nitro-node-dev --parent-chain.connection.url=<YourParentChainUrl> --chain.id=<YourOrbitChainId> --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=*
+To do that, open your `nodeConfig.json` file again, and remove the `"dangerous"` section you created before.
+
+After that, you'll have, again, two ways of running your node.
+
+**1. Using the docker-compose file**
+
+As mentioned before, this is the recommended way if you are running your orbit chain locally through the provided docker-compose file. In `docker-compose.yml`, modify the docker image used for the nitro container. Notice that we will now use the `custom-nitro-node-dev` you just created.
+
+```
+...
+nitro:
+  image: custom-nitro-node-dev
+  ports:
+...
 ```
 
-Note that `--node.staker.dangerous.without-block-validator` has been removed from this invocation now that fraud proofs are working again.
+And run `docker compose up` to run all your containers.
+
+**2. Use `docker run` to run your nitro node only**
+
+This method will only run the customized nitro node (i.e., it will not run blockscout, or the DA server if you're using an AnyTrust chain). Use the following command:
```

Comment:
```suggestion
This method will only run the customized Nitro node (i.e., it will not run Blockscout, or the DA server if you're using an AnyTrust chain). Use the following command:
```